src/seedu/addressbook/parser/Parser.java

```diff
@@ -42,7 +42,7 @@
      */
     public static final Pattern BASIC_COMMAND_FORMAT = Pattern.compile("(?<commandWord>\\S+)(?<arguments>.*)");
 
-    public Parser() {}
+    static public Parser() {}
```

Comment:
I think you may have misunderstood the instructions. You may wish to reattempt after reading this again : https://github.com/nus-cs2103-AY1617S1/addressbook-level2/blob/master/doc/LearningOutcomes.md#use-class-level-members-lo-classlevel